build.gradle

```diff
@@ -63,6 +63,7 @@ dependencies {
 
     androidTestImplementation 'junit:junit:4.12'
     androidTestImplementation "com.android.support:support-annotations:${versions.supportLibrary}"
+    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
```

Comment:
Extract version to project build.gradle?

Reply:
Currently, only versions that are used multiple times are extracted (`kotlin`, `supportLibrary` and `atsl`), I wanted to keep consistent with that.

Reply:
Before we merge this, @JakeWharton do you have any problem with adding espresso?

Reply:
Yep. It's a bit overkill though as we could inspect the properties of the returned `Toast` object.